tests/devices/qubit_mixed/test_qubit_mixed_sampling.py

```diff
@@ -0,0 +1,615 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
We shoud use the new seed architecutre instead of hardcode

Reply:
OK never mind it's fine here since we eventually called the qubit sampling probs so there's no need for testing tehse